runtime/src/test/java/com/flipkart/flux/resource/ClientElbResourceTest.java

```diff
@@ -0,0 +1,113 @@
+package com.flipkart.flux.resource;
+
+import com.flipkart.flux.api.ClientElbDefinition;
+import com.flipkart.flux.client.model.Task;
+import com.flipkart.flux.config.TaskRouterUtil;
+import com.flipkart.flux.deploymentunit.DeploymentUnit;
+import com.flipkart.flux.domain.ClientElb;
+import com.flipkart.flux.representation.ClientElbPersistenceService;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import javax.ws.rs.core.Response;
+
+import static org.mockito.Mockito.*;
+
+/**
+ * @author akif.khan
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class ClientElbResourceTest {
+
+    ClientElbPersistenceService clientElbPersistenceService = mock(ClientElbPersistenceService.class);
+
+    ClientElbResource clientElbResource = new ClientElbResource(clientElbPersistenceService);
+
+    @Test
+    public void testCreateClientElb() {
+
+        ClientElbDefinition clientElbDefinition = new ClientElbDefinition(
+                "id1", "http://10.3.2.3/api/resource/");
```

Comment:
Client Elb can't contain path in the url, it should be just plain elb address. Add this validation to the base class as well.

Reply:
Added check using java.net.URL.getPath() method which checks whether path exists in input
URL.